src/app/containers/ArticleTimestamp/shouldDisplayLastUpdatedTimestamp.test.jsx

```diff
@@ -0,0 +1,137 @@
+import shouldDisplayLastUpdatedTimestamp, {
+  hasBeenUpdated,
+  publishedAndUpdatedToday,
+} from './shouldDisplayLastUpdatedTimestamp';
+import { timestampGenerator, sameDayTimestampsGenerator } from './testHelpers';
+
+describe('shouldDisplayLastUpdatedTimestamp functions', () => {
+  let originalDate;
+
+  beforeEach(() => {
+    originalDate = Date.now;
+  });
+
+  afterEach(() => {
+    Date.now = originalDate;
+  });
+
+  it('hasBeenUpdated should return true when the time difference between firstPublished and lastPublished in minutes is greater than the minutes tolerance', () => {
+    const currentTime = originalDate();
+    const twoMinutesAgo = timestampGenerator({ minutes: 2 });
+
+    const msDifference = currentTime - twoMinutesAgo;
+    const timeDifferenceMinutes = msDifference / 1000 / 60;
+    const minutesTolerance = 1;
+
+    expect(hasBeenUpdated({ timeDifferenceMinutes, minutesTolerance })).toEqual(
+      true,
+    );
+  });
+  it('hasBeenUpdated should return false when the time difference between firstPublished and lastPublished in minutes is less than the minutes tolerance', () => {
+    const currentTime = originalDate();
+    const fortyEightSecondsAgo = timestampGenerator({ minutes: 0.8 });
+
+    const msDifference = currentTime - fortyEightSecondsAgo;
+    const timeDifferenceMinutes = msDifference / 1000 / 60;
+    const minutesTolerance = 1;
+
+    expect(hasBeenUpdated({ timeDifferenceMinutes, minutesTolerance })).toEqual(
+      false,
+    );
+  });
+
+  it(`publishedAndUpdatedToday should return true when firstPublished and lastPublished today`, () => {
+    const [midnightToday, oneAmToday] = sameDayTimestampsGenerator({
+      intervals: [{ hours: 1 }],
+    });
+
+    const wasPublishedAndUpdatedToday = publishedAndUpdatedToday({
+      firstPublished: midnightToday,
+      lastPublished: oneAmToday,
+    });
+    expect(wasPublishedAndUpdatedToday).toEqual(true);
+  });
+
+  it('publishedAndUpdatedToday should return false when firstPublished is not today', () => {
+    const oneDayAndThreeMinutesAgo = timestampGenerator({
+      days: 1,
+      minutes: 3,
+    });
+    const oneDayAndOneMinuteAgo = timestampGenerator({ days: 1, minutes: 1 });
+
+    const wasPublishedAndUpdatedToday = publishedAndUpdatedToday({
+      firstPublished: oneDayAndThreeMinutesAgo,
+      lastPublished: oneDayAndOneMinuteAgo,
+    });
+
+    expect(wasPublishedAndUpdatedToday).toEqual(false);
+  });
+  it('shouldDisplayLastUpdatedTimestamp is true when article was published and updated today', () => {
```

Comment:
Same as above - 
```
describe('shouldDisplayLastUpdatedTimestamp', () => {
it(should do X) {}

it(should do Y) {}
}
```

Reply:
Yo cheers for this. This syntax makes A LOT of sense, really like it. Gonna mark all as resolved.